contribs/gnofaucet/github.go

```diff
@@ -56,12 +68,26 @@ func (c *githubCfg) RegisterFlags(fs *flag.FlagSet) {
 	)
 }
 
+type ghFetcherCfg struct {
+	rootCfg       *serveCfg
+	fetchInterval time.Duration
+}
+
+func (c *ghFetcherCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.DurationVar(
+		&c.fetchInterval,
+		"fetch-interval",
+		20*time.Second,
```

Comment:
I would be even more conservative with this and just set it to 1 minute / 5 minutes. We don't have so many contributions across repos that we need to scrape every 20s